### PR TITLE
To Do List: add & remove

### PR DESCRIPTION
**Pull request by manzitresor**
### To Do List: add & remove
In this project, you will implement the CRUD (create, read, update, delete) methods. All the elements of the user interface will be fully functional and your application will be completed.
### Implemented Feature 

- [ ] Remove all hardcoded items from the tasks array.
- [ ] Create a new JavaScript file for the new functionality.
- [ ] Implement a function for adding a new task (add a new element to the array).
- [ ] Implement a function for deleting a task (remove an element from the array).
- [ ] Implement a function for editing task descriptions.
- [ ] Deleting a task should update all remaining items' indexes
- [ ] All changes to the To Do List should be saved in local storage.
- [ ] By default new tasks should have the property completed set to false

### CODE REVIEW

File: src/functionality.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  this.completed = completed;
+}
+
+const todotasks = localStorage.getItem('todotasks');
+const taskarr = todotasks ? JSON.parse(todotasks) : [];
+// Adding task
+function addtask() {
+  const description = descr.value;
+  const tasks = new Tasks(taskarr.length, description, false);
+  taskarr.push(tasks);
```
Comment (codepantha) [inline src/functionality.js]: - [ ] According to the project requirement, it is expected that the property index is set to the new array length. This means that if you have 5 items in your to-do list, the fifth item's index should be 5.
See the image below to understand better 👇 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @manzitresor 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Implemented Add, Remove, and Edit functionality 📌 
✔️ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by manzitresor** [comment]
Hi @codepantha Thank you so much for feedback!!
I will take that into considaration and change it.

**Review by victorokoroji — APPROVED** [review]
Hi @manzitresor, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**